dashboard/app/views/shared/_school_info.html.haml

```diff
@@ -89,9 +89,9 @@
 
   $(document).ready(function() {
     var options = #{existing_school_info.to_json};
-    options['suppressScrolling'] = #{defined? suppress_scrolling && suppress_scrolling}
+    options['suppressScrolling'] = #{!!(defined?(suppress_scrolling) && suppress_scrolling)}
 
-    if (#{defined? assume_usa && assume_usa}) {
+    if (#{!!(defined?(assume_usa) && assume_usa)}) {
```

Comment:
This is more style recommendation than anything else, so feel free to ignore as you see fit.

Insofar as we have JS inside of haml (something we'd like to get further away from), I think it would be great to have the embedded ruby as separate as possible from the JS, so something like:

```js
var suppressScrolling = #{!!(defined?(suppress_scrolling) && suppress_scrolling)};
var assumeUsa = #{!!(defined?(assume_usa) && assume_usa)};
var options = #{existing_school_info.to_json};
options.suppressScrolling = suppressScrolling;

if (assumeUsa) {
  ...
}
```